quickfixj-core/src/test/java/quickfix/SessionSettingsTest.java

```diff
@@ -313,6 +324,57 @@ public void testVariableInterpolationWithProps() throws Exception {
                 settings.getString("VariableTest"));
     }
 
+    @Test
+    public void testVariableInterpolationWithCustomPropsForSessionIdFromInputStream() throws Exception {
+        // GIVEN
```

Comment:
These comments are kind of unnecessary IMO.

Reply:
Fixed